content/en/news/release-notes.md

```diff
@@ -33,6 +33,15 @@ Fixes:
 * [OSSMC cannot update namespace or create Istio objects](https://github.com/kiali/openshift-servicemesh-plugin/issues/330)
 * [Validations - KIA0106 False Positive - Unable to Find Service Accounts](https://github.com/kiali/kiali/issues/7481)
 
+Deprecations:
+* Kiali is deprecating its current namespace selection approach. For a description of the new mechanism see https://github.com/kiali/kiali/tree/master/design/KEPS. The following configuration is deprecated:
```

Comment:
what is this link?

Reply:
That is the KEP folder where we put public designs.  We can create a specific folder for this and change the link to the more specific location.